.github/workflows/test.yml

```diff
@@ -37,12 +37,15 @@ on:
         - 3.3.0
       java:
         description: 'The Java version of Spark image.'
-        default: 11
+        default: "11"
```

Comment:
Is it neccessary?

Reply:
Not really. Value is defined as string and my linter was complaining that it's not a string. GH actions don't really care about this.